packages/flutter/lib/src/material/flexible_space_bar.dart

```diff
@@ -97,6 +100,12 @@ class FlexibleSpaceBar extends StatefulWidget {
   /// Typically an [Image] widget with [Image.fit] set to [BoxFit.cover].
   final Widget? background;
 
+  /// Shown behind the [title] when expanded.
+  ///
+  /// Compared to `background`, can build background widget more flexibly.
```

Comment:
I am not sure what this documentation means. Why would the user want to use this?